helper/src/configpresets/baselines.json

```diff
@@ -126,24 +152,35 @@
                             "cluster": {
                                 "enable_aad": true,
                                 "AksDisableLocalAccounts": true,
-                                "apisecurity": "private",
+                                "apisecurity": "none",
                                 "autoscale": true,
+                                "agentCount": 2,
+                                "maxCount": 5,
                                 "upgradeChannel": "none",
-                                "DefenderForContainers": true
+                                "DefenderForContainers": true,
+                                "AksPaidSkuForSLA": true,
+                                "SystemPoolType": "Standard",
+                                "availabilityZones": "yes"
                             },
                             "addons": {
                                 "networkPolicy": "azure",
                                 "registry": "Premium",
                                 "azurepolicy": "audit",
-                                "ingress": "appgw",
+                                "azurePolicyInitiative": "Restricted",
+                                "ingress": "nginx",
+                                "appgwKVIntegration": false,
                                 "monitor": "aci",
                                 "csisecret": "akvNew",
-                                "appgwKVIntegration": true
+                                "acrUntaggedRetentionPolicy": 15,
+                                "acrUntaggedRetentionPolicyEnabled": true,
+                                "enableACRTrustPolicy": true
                             },
                             "net": {
                                 "vnetprivateend": true,
-                                "vnet_opt": "byo",
-                                "afw": false
+                                "vnet_opt": "custom",
+                                "afw": true,
+                                "azureFirewallSku": "Premium",
+                                "nsg": true
```

Comment:
doesn't the regulated sample also do nsgflow logs?

Reply:
Yes. fixed this ty.